src/invidious/videos/parser.cr

```diff
@@ -47,6 +47,8 @@ def parse_related_video(related : JSON::Any) : Hash(String, JSON::Any)?
     "view_count"       => JSON::Any.new(view_count || "0"),
     "short_view_count" => JSON::Any.new(short_view_count || "0"),
     "author_verified"  => JSON::Any.new(author_verified),
+    "published"        => JSON::Any.new(published || ""),
+    "publishedText"    => JSON::Any.new(translate(locale, "`x` ago", recode_date(video.published, locale)) || ""),
```

Comment:
The translations should be handled on the API side. The `locale` variable never reaches the parser's scope.

Reply:
That was my suggestion. Oops. I'm surprised the CI didn't catch that though